apps/dead_letter/lib/dead_letter/server.ex

```diff
@@ -114,7 +114,7 @@ defmodule DeadLetter.Server do
   defp add_dead_letter_count(dataset_id, reason) do
     [
       dataset_id: dataset_id,
-      reason: reason
+      reason: Kernel.inspect(reason)
```

Comment:
Kernel isn't needed here. Anything in Kernel is available like `inspect(reason)`